apps/emqx_connector/src/emqx_connector_mysql.erl

```
@@ -224,7 +224,10 @@ on_get_status(_InstId, #{pool_name := PoolName} = State) ->
                     {connected, NState};
                 {error, _Reason} ->
                     %% do not log error, it is logged in prepare_sql_to_conn
-                    connecting
+                    connecting;
+                {undefined_table, NState} ->
```

Nit: 

```suggestion
                {error, {undefined_table, NState}} ->
```

(will require reordering with the catch-all clause above)